packages/uni_ui/lib/card_timeline.dart

```diff
@@ -0,0 +1,77 @@
+import 'package:flutter/material.dart';
+
+class TimelineItem extends StatelessWidget {
+  const TimelineItem(
+      {required this.startTime,
+      required this.endTime,
+      required this.card,
+      this.isActive = false,
+      super.key});
+
+  final String startTime;
+  final String endTime;
```

Comment:
I think these are not the best var names since in exams page is not a time but a date. Maybe tittle and headline, or something like that

Reply:
Okay, but we can extract the start time and end time from the exam's date and duration